src/app/pages/home-page/offer-item-page/offer-item-page.component.html

```diff
@@ -1,38 +1,37 @@
-<div *ngIf="offer$ | async as offer" class="item">
+<div *ngIf="(isLoading$ | async) === false" class="item">
   <app-offer-item [offer]="offer"></app-offer-item>
 
   <mat-tab-group mat-stretch-tabs class="tab">
-    <mat-tab label="description">
+    <mat-tab label="{{ 'CARD.DESCRIPTION' | translate }}">
       <div class="item__header">
-        <div class="item__date-end">valid to {{ offer.dateEnd }}</div>
+        <div class="item__date-end">
+          {{ 'CARD.VALID_TO' | translate }} {{ offer.dateEnd }}
+        </div>
         <div class="item__actions">
           <button
             *ngIf="offer.promocode"
             mat-button
-            class="button"
+            class="button button_promocode"
             [cdkCopyToClipboard]="offer.promocode"
           >
             {{ offer.promocode }}
             <mat-icon class="item__copy-icon">content_copy</mat-icon>
           </button>
-          <button mat-stroked-button class="button button--preorder">
-            pre-order
+          <button mat-stroked-button class="button button_preorder">
+            {{ 'CARD.PRE-ORDER' | translate }}
           </button>
         </div>
       </div>
-
       <p>
-        {{
-          offer.description || "Sorry, the brand didn't provide a description"
-        }}
+        {{ offer.description || 'CARD.DESCRIPTION_TEXT' | translate }}
       </p>
       <div class="item__footer">
         <div class="item__rate">
-          <div class="item__text">Did you use the offer? Rate it!</div>
-          <button mat-flat-button class="button button--review">review</button>
+          <div class="item__text">{{ 'CARD.RATE' | translate }}</div>
+          <button mat-flat-button class="button button_review">review</button>
         </div>
         <div class="item__favorite">
-          <div>Add to bookmarks</div>
+          <div>{{ 'CARD.BOOKMARKS_TEXT' | translate }}</div>
```

Comment:
In translate file you have CARD.BOOKMARK key, so english text is lost on page

Reply:
thanks, it's my mistake